### PR TITLE
feature(website): update navigation bar

### PR DESCRIPTION
**Pull request by svrmnk**
At this pull, I fixed the bug with transparency sub-items, but I am stuck with the popup of languages.

I consider, that this solition is the most suitable, when JS is crashed or disabled:
add "id" to popup component, than make button by tag with href="id" and using css pseudo-class :target to add styles at popup.

Will this solution suit?

### CODE REVIEW

**Review by renestalder — CHANGES_REQUESTED** [review]
@svrmnk Thank you so much for your contribution.

I don't think we are right there in terms of the right solution. While this works great without JavaScript, it removes all the ARIA-handling for screenreader users. Furthermore, the navigation can't be opened with the keyboard anymore.

We have to make sure that:

- Opening the navigation is properly announced to assistive technology. That involves using the correct aria attributes to mark what element the button is controlling and whether it is expanded/pressed. Until now, the popover components handled this. Due to removing these, those aria handlers are missing.
- That I can open it with the keyboard.

File: website/src/components/navbar/navbar-client.tsx [hunk 1]
```diff
@@ -119,14 +116,13 @@ export function NavbarClient({
...
 										) : (
-											<Popover key={index} openDelay={100} closeDelay={200}>
-												<PopoverTrigger asChild>
-													<Button variant="ghost" className="flex items-center space-x-2 py-6">
-														<Typography size="xl">{section.title}</Typography>
-														{(section.links?.length ?? 0) > 0 && <ChevronDownIcon className="h-4 w-4" />}
-													</Button>
-												</PopoverTrigger>
-												<PopoverContent asChild alignOffset={20} className="bg-popover w-56 p-0">
+											<div key={index} className="group relative">
```
[the remark this answers is not in the document]
Comment (svrmnk) [reply website/src/components/navbar/navbar-client.tsx]: I've added ability to open the submenu using the keyboard, but I don't know which ARIA was added via popover before this. So, I suggest my solition as it is now. Your team or others contributers will be able to add the required ARIA in the future or in this pull request.

[the remark this answers is not in the document]
Comment (svrmnk) [reply website/src/components/navbar/navbar-client.tsx]: @renestalder, could you please give feedback? Can we solve the problem with ARIA attributes in the following tasks? And now focus on the work of menu elements without JS in this task?

Or could yo describe in detail what needs to be done, which ARIA attributes need to be added in order for the result to satisfy you?

[the remark this answers is not in the document]
Comment (svrmnk) [reply website/src/components/navbar/navbar-client.tsx]: @renestalder, hey, could you give some feedback?

**Pull request closed without merging**